### DISCUSSION TITLE
Bug reports broken?

### DISCUSSION THREAD

**Discussion by cayhorstmann** [joaotavora/eglot#1437] [Q&A]
I tried reporting a bug with https://github.com/joaotavora/eglot/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=, and got error messages "Issue templates and forms are currently unavailable. Please try again later." and "Comment is too long (maximum is 65536 characters)" and " There was an error creating your Issue: body is too long (maximum is 65536 characters). ". No, they weren't that long.

**Comment by joaotavora** [comment]
I've just tried with a not-65536-character issue and it worked ok.

**Comment by cayhorstmann** [comment]
Oops, my bad. I didn't realize how voluminous the eglot events were. Sorry about that. I submitted to ***@***.*** instead. Should I resubmit without them, or is bug-gnu-emacs ok?

On 05/08/2024 18.08, João Távora wrote:
> I've just tried with a not-65536-character issue and it worked ok.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/joaotavora/eglot/discussions/1437#discussioncomment-10244948>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADJQO7IKPEWFWJ72HH727TZP6PRJAVCNFSM6AAAAABMARIYZSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRUGQ4TIOA>.
> You are receiving this because you authored the thread.Message ID: ***@***.***>
> 

-- 

--

Cay S. Horstmann | https://horstmann.com

**Reply by joaotavora** [reply]
bug-gnu-emacs is ok
